### PR TITLE
Adding -P [--profile] command argument

### PR DESCRIPTION
**Pull request by daniebker**
Implements #1 

Test locally using

`ruby -Ilib .\bin\aws_log_cleaner -p -P [profile] -l "test" -r eu-west-1`

### CODE REVIEW

File: lib/aws_log_cleaner/aws_credentials.rb [hunk 1]
```diff
@@ -9,17 +9,58 @@ class AwsCredentials
...
       @credentials =
         if key_id.nil? && secret.nil?
-          Aws::SharedCredentials.new
+          begin
+            profile.nil? ? Aws::SharedCredentials.new : Aws::SharedCredentials.new(profile_name: profile)
+          rescue Aws::Errors::NoSuchProfileError
+            raise_credentials_error
+          end
         else
           Aws::Credentials.new(key_id, secret)
```
Comment (betabandido) [inline lib/aws_log_cleaner/aws_credentials.rb]: Can this throw? If so we should use a rescue block here too.

File: lib/aws_log_cleaner/aws_credentials.rb [hunk 2]
```diff
@@ -9,17 +9,58 @@ class AwsCredentials
...
+      return @credentials
+    end
+
+    private
+
+    def raise_credentials_error
+      raise "Could not load credentials."
+    end
+
+    def parse_credential_arguments(credential_args)
```
Comment (betabandido) [inline lib/aws_log_cleaner/aws_credentials.rb]: Based on the method name, I do not expect that `@region` is set here. I think to be more consistent, we could just return `region` too, and then pass it to `set_credentials` as well.

File: lib/aws_log_cleaner/aws_credentials.rb [hunk 3]
```diff
@@ -9,17 +9,58 @@ class AwsCredentials
...
+      if credential_args.nil?
+        key_id  = ENV['AWS_ACCESS_KEY_ID']
+        secret  = ENV['AWS_SECRET_ACCESS_KEY']
+        @region = ENV['AWS_DEFAULT_REGION']
```
Comment (betabandido) [inline lib/aws_log_cleaner/aws_credentials.rb]: I think we should look first for `AWS_REGION`, as I think it's the most common name. If I remember correctly `AWS_DEFAULT_REGION` is an alias.

File: lib/aws_log_cleaner/aws_credentials.rb (hunk 2 above)
Comment (daniebker) [inline lib/aws_log_cleaner/aws_credentials.rb]: good point

File: lib/aws_log_cleaner/aws_credentials.rb (hunk 3 above)
Comment (daniebker) [inline lib/aws_log_cleaner/aws_credentials.rb]: ok.

File: lib/aws_log_cleaner/aws_credentials.rb (hunk 1 above)
Comment (daniebker) [inline lib/aws_log_cleaner/aws_credentials.rb]: Not sure. I'll try and add a test.

**Pull request merged**